web/main/urls.py

```diff
@@ -6,127 +6,330 @@
 from django.views.generic import RedirectView, TemplateView
 from rest_framework.urlpatterns import format_suffix_patterns
 
-from .models import Casebook, Section, ContentNode, ContentAnnotation, LegalDocument, SavedImage
+from .models import (
+    Casebook,
+    Section,
+    ContentNode,
+    ContentAnnotation,
+    LegalDocument,
+    SavedImage,
+)
 from .test.test_permissions_helpers import no_perms_test
-from .url_converters import IdSlugConverter, OrdinalSlugConverter, register_model_converter
+from .url_converters import (
+    IdSlugConverter,
+    OrdinalSlugConverter,
+    register_model_converter,
+)
 from .utils import fix_after_rails
 from . import views, forms
 
 
-register_converter(IdSlugConverter, 'idslug')
-register_converter(OrdinalSlugConverter, 'ordslug')
+register_converter(IdSlugConverter, "idslug")
+register_converter(OrdinalSlugConverter, "ordslug")
 register_model_converter(Casebook)
 register_model_converter(Section)
-register_model_converter(ContentNode, 'resource')
-register_model_converter(ContentAnnotation, 'annotation')
+register_model_converter(ContentNode, "resource")
+register_model_converter(ContentAnnotation, "annotation")
 register_model_converter(LegalDocument)
-register_model_converter(SavedImage,name='image_uuid', base=UUIDConverter, field='external_id')
+register_model_converter(
+    SavedImage, name="image_uuid", base=UUIDConverter, field="external_id"
+)
 
 # these patterns will have optional format suffixes added, like '.json'
```

Comment:
Do we want to `fmt: off` for the stanzas declaring paths? I feel like it is significantly easier to read if you can skim the list for the route you are looking for, without being interrupted by other args when things get long, like in the reformatted
```
    path(
        "resources/<resource:resource>/annotations/<annotation:annotation>",
        views.AnnotationDetailView.as_view(),
        name="annotation_detail",
    ),
```

Reply:
Personally I don't customize `black`'s output any and just let it do what it will, so no opinion here from me!

Reply:
🤷‍♀️ If that's the best practice, okey dokey. This may just be something for me to get used to 🙂

Reply:
Would we be willing to give 100 a try at first? Can easily switch subsequently to the default 88 if its grinding people's gears. 

But seriously, don't let me be a pain in the butt here 😁

Reply:
Totally!